components/ProfilePage/Deploy6551AndMintButton.tsx

```diff
@@ -0,0 +1,29 @@
+import { Contract } from "ethers"
+import { Button } from "../../shared/Button"
+import abi from "../../lib/abi-dna-minter.json"
+import { useProfileProvider } from "../../providers/ProfileContext"
+import { useEthersSigner } from "../../hooks/useEthersSigner"
+
+const Deploy6551AndMintDNAButton = () => {
+  const { cre8orNumber } = useProfileProvider()
+  const signer = useEthersSigner({ chainId: process.env.NEXT_PUBLIC_TESTNET ? 5 : 1 })
+
+  const onClick = async () => {
+    const dnaMinter = "0xeBBe9F31ef46ae687a60eB25F38eF8B8b2311E25"
```

Comment:
is this supposed to be hardcoded contract address? We usually set them in env variables.

Reply:
good catch, setting this to `NEXT_PUBLIC_DNA_MINTER` 👍